tests/utils/test_raster.py

```diff
@@ -241,3 +273,40 @@ def test_sum_resample_file(self, resample_xarr_fpath: str) -> None:
         # assert correct resampling values (summing consitiuent grids)
         expected_result = np.array([[[14, 22], [46, 54]]])
         assert np.array_equal(expected_result, xds_out.to_numpy())
+
+    @pytest.mark.parametrize(
```

Comment:
Are we able to include the 'match' parameter of pytest.raises() here?  
This will ensure that the tests are detecting the correct errors, and not passing on other errors of the same type.

Reply:
Good idea. This one is now captured in a738242. Happy to close this conversation if you are too.